SECURITY.md

```diff
@@ -0,0 +1,61 @@
+# Security policy
+
+## Charmcraft and charms
+
+Charmcraft is a tool to create and publish charms. A charm is a software artifact that
+contains instructions to manage a separate application in the context of a [Juju] model,
```

Comment:
possibly we should also mention things like resources but I'm not sure I know enough to write something decent

Reply:
Not sure we should bother - the only thing Charmcraft really does with resources is upload them and do basic store operations.